backend/src/routes/recipes.js

```diff
@@ -4,9 +4,56 @@ const router = express.Router();
 const queries = require("../prisma/recipes");
 
 /* GET recipes listing. */
+// todo
 router.get('/', async function (req, res, next) {
-  const result = await queries.getRecipes();
-  res.json(result)
+  try {
+    const result = await queries.getRecipes();
+    res.json(result)
+  } catch (e) {
+    res.status(500);
+    res.send("Error in get recipes.");
+  }
+});
+
+/* CREATE recipe. */
+router.post('/', async function (req, res, next) {
+  try {
+    const result = await queries.createRecipe(req.body);
+    res.json(result);
+  } catch (e) {
+    res.status(500);
+    res.send("Error in create recipe.");
+  }
+});
+
+
+/* DELETE recipe. */
+router.delete('/:id', async function (req, res, next) {
+  console.log(req.params.id);
+  try {
+    await queries.deleteRecipe(parseInt(req.params.id));
+    res.status(204);
+    res.send("Recipe id " + req.params.id + " was deleted.")
+  } catch (e) {
```

Comment:
Same thing, speficic error needs to be matched.
https://realpython.com/the-most-diabolical-python-antipattern/
Different language, same antipattern.

Reply:
todo - smazat z errors